smartsim/entity/__init__.py

```diff
@@ -27,7 +27,7 @@
 from .application import Application
 from .dbnode import FSNode
 from .dbobject import *
-from .ensemble import Ensemble
+from .._core.builders.ensemble import Ensemble
```

Comment:
I think ``_core`` is mostly reserved for non-user infrastructure.  I think maybe ``builders`` should be in top-level SmartSim.  Thoughts?

Reply:
I believe that strategies.py might also need to be moved, am I correct on this?

Reply:
strategies moved